src/connection.ts

```diff
@@ -290,6 +291,17 @@ interface AzureActiveDirectoryServicePrincipalSecret {
   };
 }
 
+/** Structure that defines the options that are necessary to authenticate the tedious instance with an `@azure/identity` chained token credential derived class. */
+interface CredentialChainAuthentication {
+    /** Unique designator for the type of authentication to be used. */
```

Comment:
I think this is still using tabs instead of spaces. 😬

Reply:
Sorry about that, I tried to regex search `/t` in GitHub but I found nothing. Should be fixed now.

Reply:
@arthurschreiber, I found one other place where the formatting might not be correct. Fixed that too.

Reply:
In the future, I will 100% not be free handing this stuff in the GitHub file editor. I will follow the PR instructions and do it locally :-P